benchmarks/01_trivial/05_packed_load_store.ispc

```diff
@@ -0,0 +1,63 @@
+                num += packed_load_active(&src[num], &var);                                                            \
+            dst[i] = var;                                                                                              \
+        }                                                                                                              \
+        return programCount;                                                                                           \
+    }
+
+// This case has mask set all_on.
+#define PACKEDSTORE_ALL(T_ISPC)                                                                                        \
+    export uniform int32 packed_store_##T_ISPC##_all(uniform T_ISPC *uniform src, uniform T_ISPC *uniform dst,         \
+                                                     uniform int count) {                                              \
+        uniform int32 num = 0;                                                                                         \
+        foreach (i = 0...count) {                                                                                      \
+            varying T_ISPC var = src[i];                                                                               \
+            num += packed_store_active(&dst[num], var);                                                                \
+        }                                                                                                              \
+        return programCount;                                                                                           \
+    }
+
+// This case stores one value per loop.
+#define PACKEDSTORE_ONE(T_ISPC)                                                                                        \
+    export uniform int32 packed_store_##T_ISPC##_one(uniform T_ISPC *uniform src, uniform T_ISPC *uniform dst,         \
+                                                     uniform const int index, uniform int count) {                     \
+        uniform int32 num = 0;                                                                                         \
+        foreach (i = 0...count) {                                                                                      \
+            varying T_ISPC var = src[i];                                                                               \
+            if (programIndex == index)                                                                                 \
+                num += packed_store_active(&dst[num], var);                                                            \
+        }                                                                                                              \
+        return programCount;                                                                                           \
+    }
+
+PACKEDLOAD_ALL(int32)
+PACKEDLOAD_ONE(int32)
+PACKEDSTORE_ALL(int32)
```

Comment:
We need `packed_store_active2()` next to `packed_store_active()`.

Reply:
Added